backend/shork/src/test/kotlin/memory/TestMemoryCore.kt

```diff
@@ -98,4 +98,99 @@ internal class TestMemoryCore {
         val writes = gameDataCollector.getGameStatistics().map { it.memoryWrites }.flatten()
         assertEquals(listOf(42, 1337, 666), writes)
     }
+
+    fun testResolveForReadBoundsPositive() {
+        val defaultInstruction =
+            MockInstruction(42, 69, AddressMode.DIRECT, AddressMode.DIRECT, Modifier.I)
+        val memoryCore10 =
+            MemoryCore(100, getDefaultInternalSettings(defaultInstruction, readDistance = 10))
+
+        val out = memoryCore10.resolveForReading(42, 23, AddressMode.DIRECT)
+        assertEquals(45, out)
+
+        val out2 = memoryCore10.resolveForReading(42, 34, AddressMode.DIRECT)
+        assertEquals(46, out2)
+
+        val memoryCore32 =
+            MemoryCore(256, getDefaultInternalSettings(defaultInstruction, readDistance = 32))
+
+        for (i in 0 until 100) {
+            for (j in 0 until 32) {
+                val out = memoryCore32.resolveForReading(i, j, AddressMode.DIRECT)
+                assertEquals(i + j % 32, out)
+            }
+        }
+    }
```

Comment:
Is there an @Test missing above this function? Like this it wont be run when running the tests.